apps/meteor/app/reactions/server/setReaction.ts

```diff
@@ -100,24 +78,32 @@ async function setReaction(room: IRoom, user: IUser, message: IMessage, reaction
 		await Messages.setReactions(message._id, message.reactions);
 		if (isTheLastMessage(room, message)) {
 			await Rooms.setReactionsInLastMessage(room._id, message.reactions);
-			void notifyOnRoomChangedById(room._id);
 		}
-		await callbacks.run('setReaction', message._id, reaction);
```

Comment:
If we are getting rid of this callback, can we remove it from the callback's definition as well?